lib/ansible/modules/stat.py

```diff
@@ -468,6 +468,8 @@ def main():
     try:
         if follow:
             st = os.stat(b_path)
+            b_path = os.path.realpath(b_path)
+            path = to_native(b_path, errors='strict')
```

Comment:
this is incorrect, the path should still show the link itself, see `link_source` for the destination path.

Reply:
I think i got misleading because of the `follow` arg behaviour on file or find module 🤔. I close.